src/compiler/scala/tools/nsc/tasty/bridge/TastyCore.scala

```diff
@@ -0,0 +1,38 @@
+package scala.tools.nsc.tasty.bridge
+
+import scala.tools.nsc
+import nsc.symtab, nsc.tasty.TastyUniverse
+
+abstract class TastyCore { self: TastyUniverse =>
+  import self.{symbolTable => u}
```

Comment:
Ideally this should extend some version where all the types and operations are abstract and thats what TreeUnpickler sees, but perhaps that would be useless, depending on what shared API is extracted for reading TASTy.